handbook/company/pricing-features-table.yml

```diff
@@ -941,7 +916,8 @@
   usualDepartment: IT
   tier: Premium
   comingSoonOn: 2024-12-31 #customer-rosner
-- industryName: Automated user role sync via Okta, AD, or any IDP
+- industryName: User account sync
```

Comment:
Renamed to "User account sync" and moved the specifics into the description/tooltip.